www/main/settings/utils/apply-theme.js

```diff
@@ -0,0 +1,30 @@
+import { remote } from 'electron';
+
+const { store } = remote.require('./app');
+const analytics = remote.getGlobal('analytics');
+
+export const applyTheme = (themeInstance, isCustom, setTheme, setThemeBg) => {
+  if (!isCustom) {
+    setTheme(themeInstance.key);
```

Comment:
![image](https://user-images.githubusercontent.com/2982784/113921314-b4dbe600-979a-11eb-82ac-131c93b10367.png)
Still not applying to ceremonies "current theme" option

Reply:
Oops sorry this is fixed, pushing commmit now.. :(